tools/clang/include/clang/SPIRV/ModuleBuilder.h

```diff
@@ -68,7 +68,9 @@ class ModuleBuilder {
 
   /// \brief Creates a SPIR-V basic block. On success, returns the <label-id>
   /// for the basic block. On failure, returns zero.
-  uint32_t createBasicBlock(llvm::StringRef name = "");
+  /// All basic blocks are reachable by default. If isReachable is set to false,
```

Comment:
As discussed offline, it seems we do not need this `isReachable` parameter and all reachability stuff in `BasicBlock`.